### DISCUSSION TITLE
Check changes in test-bot

### DISCUSSION THREAD

**Discussion by Akylzhan** [Homebrew/discussions#45] [Everyday usage] [answered]
If I locally commit changes to test-bot, how to check them? Maybe there is an option which downloads my version of test-bot and uses it?

I am submitting this https://github.com/Homebrew/brew/pull/8851 and test-bot --only-formulae uses `brew list` instead of `brew list --formulae`. Therefore, I wanted to submit PR to add '--formulae' but don't know how to check my changes

**Comment by Rylan12** [comment, marked as answer]
I think you're looking for the [Homebrew/homebrew-test-bot](https://github.com/Homebrew/homebrew-test-bot) repository.

If you have changes to make to the brew test-bot code feel free to open a PR in that repo.

**Reply by Akylzhan** [reply]
I mean I have made changes to test-bot locally and want to check them. How to tell `brew test-bot` to use my local test bot for testing?

It is small change: 
https://github.com/Akylzhan/homebrew-test-bot/blob/87a86a31cdd984259a1fc08b746aae5a2fd002d3/lib/tests/formulae.rb#L258
Just added '--formulae' here. But still I want to make sure that it works :)

**Reply by jonchang** [reply]
`brew test-bot --test-default-formula` might work. You can also check the CI workflow, in the `.github` directory in the test-bot repository.
